File: Bars.Gkh/libs/B4/model/Dict/contractservice/CommunalContractService.js
Ext.define('B4.model.dict.contractservice.CommunalContractService', {
    extend: 'B4.model.dict.contractservice.ManagementContractService',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'communalContractService'
    },
    fields: [
        { name: 'CommunalResource' },
        { name: 'SortOrder' },
        { name: 'IsHouseNeeds', defaultValue: false }
    ]
});

File: Bars.Gkh/Entities/Dicts/ContractService/CommunalContractService.cs
namespace Bars.Gkh.Entities.Dicts.ContractService
{
    using Bars.Gkh.Enums;

    /// <summary>
    /// Коммунальная услуга
    /// </summary>
    public class CommunalContractService : ManagementContractService
    {
        /// <summary>
        /// Коммунальный ресурс
        /// </summary>
        public virtual TypeCommunalResource CommunalResource { get; set; }

        /// <summary>
        /// Порядок сортировки 
        /// </summary>
        public virtual int SortOrder { get; set; }

        /// <summary>
        /// Услуги предоставляется на ОДН (общедомовые нужды)
        /// </summary>
        public virtual bool IsHouseNeeds { get; set; }
    }
}
